dev-aws/kafka-shared-msk/account-identity/holders.tf

```
@@ -0,0 +1,24 @@
+import {
+  to = kafka_topic.account_identity_legacy_account_holder_events_compacted
+  id = "account-identity.legacy.account.holder.events.compacted"
+}
+
+resource "kafka_topic" "account_identity_legacy_account_holder_events_compacted" {
+  name               = "account-identity.legacy.account.holder.events.compacted"
+  replication_factor = 3
+  partitions         = 15
+  config = {
+    "compression.type" = "zstd"
+    "retention.ms"     = "604800000"
```

Would be great to comment with a human readable value:

```suggestion
    # keep data for 7 days
    "retention.ms"     = "604800000"
```